Source/Core/VideoBackends/Vulkan/VKTexture.cpp

```diff
@@ -802,43 +781,25 @@ std::pair<VkImage, VkDeviceMemory> VKStagingTexture::CreateLinearImage(StagingTe
     return std::make_pair(VK_NULL_HANDLE, VK_NULL_HANDLE);
   }
 
-  VkImage image;
-  res = vkCreateImage(g_vulkan_context->GetDevice(), &image_info, nullptr, &image);
-  if (res != VK_SUCCESS)
-  {
-    LOG_VULKAN_ERROR(res, "vkCreateImage failed: ");
-    return std::make_pair(VK_NULL_HANDLE, VK_NULL_HANDLE);
-  }
-
-  // Allocate memory to back this texture, we want device local memory in this case
-  VkMemoryRequirements memory_requirements;
-  vkGetImageMemoryRequirements(g_vulkan_context->GetDevice(), image, &memory_requirements);
-
-  VkMemoryAllocateInfo memory_info = {
-      VK_STRUCTURE_TYPE_MEMORY_ALLOCATE_INFO, nullptr, memory_requirements.size,
-      g_vulkan_context
-          ->GetMemoryType(memory_requirements.memoryTypeBits, VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT,
-                          false)
-          .value_or(0)};
+  VmaAllocationCreateInfo alloc_create_info = {};
+  alloc_create_info.flags = 0;
```

Comment:
Is there a reason you stuck with no flags over the suggestion for staging?  ( `VMA_ALLOCATION_CREATE_HOST_ACCESS_SEQUENTIAL_WRITE_BIT | VMA_ALLOCATION_CREATE_MAPPED_BIT` )

Reply:
We copy the linear image to the staging buffer anyway. So we don't map it.

That's technically a unnecessary copy but there's two reasons for that:
* Linear images aren't necessarily supported for a cached memory type. They *probably* are in this case but I'd have to check this and still do the copy if they aren't.
* More important reason: I wanted to keep this stupid workaround simple.

Reply:
Sorry, missed this was the workaround.  This is fine.  Thanks for your comment.